aws/resource_aws_redshift_cluster.go

```diff
@@ -219,22 +219,54 @@ func resourceAwsRedshiftCluster() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
-			"enable_logging": {
-				Type:     schema.TypeBool,
+			"logging": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
 				Optional: true,
-				Default:  false,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enable": {
+							Type:     schema.TypeBool,
+							Required: true,
```

Comment:
does this not need a State Migration?

Reply:
I don't think so - it's only required within the context of `logging` block, i.e. you can't define an empty `logging {}`, it shouldn't affect any existing configs without this block.